pkg/sfu/streamtracker.go

```diff
@@ -44,27 +48,32 @@ type StreamTrackerParams struct {
 type StreamTracker struct {
 	params StreamTrackerParams
 
-	onStatusChanged func(status StreamStatus)
+	onStatusChanged    func(status StreamStatus)
+	onBitrateAvailable func()
+
+	lock sync.RWMutex
```

Comment:
Using lock here consistently. There were too many atomics and it was getting hard to follow. All the functions here are very well scoped and extremely small. So, just use the lock consistently.